Features/Transfer/Sources/Services/ConfirmService.swift

```diff
@@ -0,0 +1,82 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+import SwapService
+import BalanceService
+import PriceService
+import TransactionService
+import ExplorerService
+import Keystore
+import Swap
+import WalletsService
+import ScanService
+import Primitives
+import PrimitivesComponents
+import BigInt
+import Blockchain
+import ChainService
+
+public struct ConfirmService: Sendable {
```

Comment:
I think it would be better to move the methods `getExplorerLink`, `getPasswordAuthentication` and mb `defaultPriority` to another place, as they seem to have a different responsibility